Cloud Application Manager/Data Center Edition/camdce-with-aws-master-account.md

```diff
@@ -9,50 +9,50 @@
 
 **In this article:**
 
-* [Overview](#overview)
-* [Audience](#audience)
-* [Prerequisites](#prerequisites)
-* [Additional information](#additional-information)
-* [Create the Policy for the EC2 Instance Role](#create-the-policy-for-the-ec2-instance-role)
-* [Create the EC2 Instance Role](#create-the-ec2-instance-role)
-* [Attach the IAM Role to the Cloud Application Manager Data Center Edition EC2 Instance](#attach-the-iam-role-to-the-cloud-application-manager-data-center-edition-ec2-instance)
-* [Enable the EC2 Instace Role on CAM](#enable-the-ec2-instace-role-on-cam)
-* [Create a Policy and Role for the managed account](#create-a-policy-and-role-for-the-managed-account)
-* [Create a Cloud Application Manager AWS Provider](#create-a-cloud-application-manager-aws-provider)
-* [Contacting Cloud Application Manager Support](#contacting-cloud-application-manager-support)
+- [Overview](#Overview)
+- [Audience](#Audience)
+- [Prerequisites](#Prerequisites)
+- [Additional information](#Additional-information)
+- [Create the Policy for the Master Account role](#Create-the-Policy-for-the-Master-Account-role)
+- [Method 1: create the user and API credentials](#Method-1-create-the-user-and-API-credentials)
+- [Method 2: create the EC2 Instance Role](#Method-2-create-the-EC2-Instance-Role)
+- [Enable the Master Account on Cloud Application Manager](#Enable-the-Master-Account-on-Cloud-Application-Manager)
+- [Create a Policy and Role for the managed account](#Create-a-Policy-and-Role-for-the-managed-account)
+- [Create a Cloud Application Manager AWS Provider](#Create-a-Cloud-Application-Manager-AWS-Provider)
 
 
 ### Overview
 
 
-This article shows how to leverage an EC2 Instance Role as an authorization mechanism for all AWS API calls from a Cloud Application Manager Data Center Edition (referred to as appliance from now on). EC2 Instance Roles ease credential management and increase security for applications running in EC2 instances. You can learn more about them in [the AWS documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html).
+This article shows how to leverage a Master Account as the only authentication principal for AWS accounts in Cloud Application Manager Data Center Edition. Once this is enabled, AWS providers are identified only by a role ARN instead of by key and secret. It is then possible then to manage AWS accounts from Cloud Application Manager without creating account-specific credentials: AWS will authenticate only the master account, and grant privileges on the managed account based on the ARN role. 
```

Comment:
```suggestion
This article shows how to leverage a Master Account as the only authentication principal for AWS accounts in Cloud Application Manager Data Center Edition. Once this is enabled, AWS providers are identified only by a role ARN instead of by key and secret. It is then possible to manage AWS accounts from Cloud Application Manager without creating account-specific credentials: AWS will authenticate only the master account, and grant privileges on the managed account based on the ARN role. 
```